manifests/a/Alibaba/aDrive/2.2.4/Alibaba.aDrive.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Alibaba.aDrive
+PackageVersion: 2.2.4
+MinimumOSVersion: 10.0.0.0
+Installers:
+- InstallerLocale: zh-CN
+  Architecture: x86
```

```suggestion
- Architecture: x86
```
It looks like we need to start to move away from using `InstallerLocale` here because this prevents machines outside of the `zh-CN` locale from upgrading the application using the `winget upgrade` command.

See the comment that @denelon wrote at https://github.com/microsoft/winget-pkgs/issues/25347#issuecomment-903016232 & https://github.com/microsoft/winget-pkgs/issues/25347#issuecomment-903033816.